Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/meat.yml

```
@@ -54,6 +54,70 @@
     cycles: 1
     output: FoodBakedNugget
 
+- type: entity
+  name: raw fish fillet
+  parent: FoodMeatBase
+  id: FoodCuredCarp
+  description: A slice of treated alien space fish. It has been rendered safe to eat. Smells like space and fish.
```

```suggestion
  description: A slice of fresh space fish, safe to eat. Smells like space and fish.
```
In case you want to grow/harvest the fish yourself, this probably makes more sense.